docs/tech-stack/linux.md

```diff
@@ -13,7 +13,7 @@ The project was founded in 1991 as a personal project by Linus Torvalds (hence t
 
 ## Which distro should I pick?
 
-In the context of building PMS the choice is kind of made for you, if you're interested at all in using ZFS or WireGuard. Ubuntu is the only mainstream distribution which ships with both projects compiled into the kernel and doesn't rely on DKMS - see *ZFS -> [What about the license?](../zfs/#what-about-the-license)*.
+In the context of building PMS the choice is kind of made for you, if you're interested at all in using ZFS or WireGuard. Ubuntu is the only mainstream distribution which ships with both projects compiled into the kernel and doesn't rely on DKMS - see *ZFS -> [What about the license?](zfs.md/#what-about-the-license)*.
```

Comment:
I think you still need the `../` in here, as well as now you can remove the trailing `/`before the `#`

Reply:
Regarding the `../` that's what was fundamentally the issue. Since both files are under the 'Tech-stack' section they do not need to use `../` to access each other. Updating the patch to remove the trailing `/`